medusa/server/api/v2/series.py

```diff
@@ -64,7 +65,8 @@ def filter_series(current):
             return self._not_found('Series not found')
 
         detailed = self._parse_boolean(self.get_argument('detailed', default=True))
-        data = series.to_json(detailed=detailed)
+        episodes = self._parse_boolean(self.get_argument('episodes', default=False))
```

Comment:
This changes the current behavior:
Currently `detailed` defaults to `True` which means episodes are added to the response.
Was this intended?

Reply:
Yes. As afaik only show-header makes use of it. And by default is like to keep the response as small as possible.

Maybe we should also change detailed by default to false.

In the displayShow we use the episodes route to get episodes per season. So this is preparation for that